classes/person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand(1..10_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    true if is_of_age? || @parent_permission
+    false
```

Comment:
- [ ] [_Optional_] You can completely remove the `true` and `false` in this check. The conditional 
``` ruby
is_of_age? || @parent_permission
```
will automatically return true or false.